backend/de.metas.contracts/src/main/java/de/metas/contracts/modular/interest/InterestComputationRequest.java

```diff
@@ -41,6 +41,7 @@ public class InterestComputationRequest
 	@NonNull Money interestToDistribute;
 	@NonNull LockOwner lockOwner;
 	@NonNull ComputingMethodType computingMethodType;
+	@NonNull Integer interestScale;
```

Comment:
shall we use CurrencyPrecision instead of Integer?
(TBH not sure what interestScale means)